tracy-edit/src/OfflineSymbolResolver.cpp

```diff
@@ -0,0 +1,136 @@
+#include <fstream>
+#include <iostream>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unordered_map>
+
+#include "../../server/TracyWorker.hpp"
+#include "../../zstd/zstd.h"
+
+#include "OfflineSymbolResolver.h"
+
+bool ApplyPathSubstitutions( std::string& path, const PathSubstitutionList& pathSubstituionlist )
+{
+    for( const auto& substituion : pathSubstituionlist )
+    {
+        if( std::regex_match(path, substituion.first) )
+        {
+            path = std::regex_replace( path, substituion.first, substituion.second );
+            return true;
+        }
+    }
+    return false;
+}
+
+tracy::StringIdx AddSymbolString( tracy::Worker& worker, const std::string& str )
+{
+    // TODO: use string hash map to reduce potential string duplication?
+    tracy::StringLocation location = worker.StoreString( str.c_str(), str.length() );
+    return tracy::StringIdx( location.idx );
+}
+
+bool PatchSymbols( tracy::Worker& worker, const PathSubstitutionList& pathSubstituionlist, bool verbose )
+{
+    uint64_t callstackFrameCount = worker.GetCallstackFrameCount();
+    std::string relativeSoNameMatch = "[unresolved]";
+
+    std::cout << "Found '" << callstackFrameCount << "' callstack frames. Batching into image groups..." << std::endl;
```

Comment:
Is the count supposed to be quoted here?

Reply:
I've remove all the numeric quoting in there